lib/fs.js

```diff
@@ -520,7 +520,6 @@ function tryStatSync(fd, isUserFd) {
   } finally {
```

Comment:
@BridgeAR please note that the performance profile of try/catch/finally changed quite a bit since the code was added most likely :)